### PR TITLE
adapt deploy scripts to stable/master branches

### PR DESCRIPTION
**Pull request by 3nids**
we will build APKs for master and stable but only master ones are pushed to play store

also increase version code for PR builds so we won't need to uninstall all the time before installing new APK

### CODE REVIEW

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-819-2ff63c84a353d4964b7621b3720d99a707f61ee0-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-819-2ff63c84a353d4964b7621b3720d99a707f61ee0-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-819-2ff63c84a353d4964b7621b3720d99a707f61ee0-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-819-2ff63c84a353d4964b7621b3720d99a707f61ee0-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-819-65e7dfe5bc265e560f7f4ff87bbaae04ed3386bf-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-819-65e7dfe5bc265e560f7f4ff87bbaae04ed3386bf-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-819-65e7dfe5bc265e560f7f4ff87bbaae04ed3386bf-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-819-65e7dfe5bc265e560f7f4ff87bbaae04ed3386bf-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-819-aa42fc5526d320904d847cd16aa94d7573a23311-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-819-aa42fc5526d320904d847cd16aa94d7573a23311-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-819-aa42fc5526d320904d847cd16aa94d7573a23311-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-819-aa42fc5526d320904d847cd16aa94d7573a23311-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-819-0a21fa40de023f596d001e425c2cdf6fb33291f3-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-819-0a21fa40de023f596d001e425c2cdf6fb33291f3-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-819-0a21fa40de023f596d001e425c2cdf6fb33291f3-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-819-0a21fa40de023f596d001e425c2cdf6fb33291f3-x86_64.apk)

File: scripts/ci/travis_build.sh [hunk 1]
```diff
@@ -30,18 +30,22 @@ elif [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then
...
-  echo "Commit number: ${COMMIT_COUNT_SINCE_v140}"
+  echo "Commit number: ${COMMIT_COUNT_SINCE_LAST_TAG}"
   echo "Arch number: ${ARCH_NUMBER}"
 
 else
   echo "Building pull request beta"
+  # use a date for the version code since it's the most working solution
+  # to switch from a PR to another without needing to uninstall
+  # (as long as you always install a more recent version)
+  CUR_DATE=$(date +%Y%j%H%M)
```
Comment (m-kuhn) [inline scripts/ci/travis_build.sh]: Can we move this to a separate PR?

Comment (3nids) [reply scripts/ci/travis_build.sh]: done

File: scripts/deploy.sh [hunk 2]
```diff
@@ -24,8 +24,8 @@ if [[ ${TRAVIS_SECURE_ENV_VARS} = true ]]; then
     echo -e "\e[31mDeploying app to pull request\e[0m"
     echo "${BODY}" | curl -u m-kuhn:${GITHUB_API_TOKEN} -X POST --data @- https://api.github.com/repos/opengisch/QField/issues/${TRAVIS_PULL_REQUEST}/comments
 
-  elif [[ -n ${TRAVIS_TAG} ]]; then
-    # we are on a standard commit on master branch => push to ALPHA
+  elif [[ -n ${TRAVIS_TAG} ]] && [[ ${TRAVIS_BRANCH} = stable ]]; then
+    # we are on a tag and on stable branch
```
Comment (m-kuhn) [inline scripts/deploy.sh]:
```suggestion
    # we are on a tag and on stable branch => push to ALPHA (play store)
```

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-819-8d39d137f9efaf730bec77650363cacd1aaa0cc0-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-819-8d39d137f9efaf730bec77650363cacd1aaa0cc0-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-819-8d39d137f9efaf730bec77650363cacd1aaa0cc0-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-819-8d39d137f9efaf730bec77650363cacd1aaa0cc0-x86_64.apk)
